README.md

```diff
@@ -2,6 +2,15 @@
 
 Office Add-ins code samples are provided in this repo to help you learn, study, and built great Office Add-ins!
 
+## Getting started
+
+The following samples show how to build the simplest Office Add-in with only a manifest, HTML web page, and a logo. They will help you understand the fundamental parts of an Office Add-in. For additional getting started information, see our [quickstarts](https://docs.microsoft.com/office/dev/add-ins/quickstarts/excel-quickstart-jquery) and [tutorials](https://docs.microsoft.com/search/?terms=tutorial&scope=Office%20Add-ins).
+
+* [Hello world Excel add-in](https://github.com/OfficeDev/PnP-OfficeAddins/tree/main/Samples/hello-world/excel-hello-world)
```

Comment:
Consider something like this, placing the Office host name first. Also maybe alphabetize.
```suggestion
* [Excel "Hello world" add-in](https://github.com/OfficeDev/PnP-OfficeAddins/tree/main/Samples/hello-world/excel-hello-world)
```

Reply:
Fixed